packages/debian/download_packages.py

```diff
@@ -176,16 +176,22 @@ def download_package(package: apt.package.Version) -> pathlib.Path:
 
 def main(packages: Sequence[str], env: Mapping[str, str]) -> None:
     """Download the packages specified on the command-line."""
+    #pathlib.Path('/repositories/.witness').unlink()
     add_external_repositories(env['SALT_VERSION'])
     apt_pkg.init()
     cache = apt.cache.Cache()
     to_download = {}
+    dest = pathlib.Path('/repositories')
     for package in packages:
         deps = get_package_deps(package, cache)
         to_download.update(deps)
     for pkg in to_download.values():
         filepath = download_package(pkg)
         os.chown(filepath, int(env['TARGET_UID']), int(env['TARGET_GID']))
+    #pathlib.Path('/repositories/.witness').touch()
+    for rep in dest.iterdir():
```

Comment:
Agree with that, but the repository folder will be created by the class DEBRepository in targets/repository.py like it's done for RedHat... so it will be handled at the next step about the repository... what do you think about it ?

Reply:
Why not, but then add a TODO like I did (`# TODO: to be done by the build chain.`)